packages/plugin-expo-app/app.js

```diff
@@ -16,23 +16,29 @@ module.exports = {
       lastState = newState
     })
 
-    let nativeBundleVersion, nativeVersionCode
+    let nativeBundleVersion, nativeVersionCode, bundleVersion, versionCode
 
-    if (Constants.appOwnership === 'standalone') {
+    if (Constants.appOwnership !== 'expo') {
```

Comment:
`Constants.appOwnership` will always be null for standalone apps built using EAS Build